### ISSUE TITLE
Decouple front-end and back-end completely?

### ISSUE THREAD

**Issue by robert-danowski** [Requarks/wiki#285]
Will there be a possibility to keep front-end and back-end on different servers? 

For example someone who works in a big international corporation behind a very strong firewall could put front-end in surge.sh (where this user has access) and this front-end would communicate with other other server.

This could allow to build native applications for mobile devices later.

**Comment by NGPixel** [comment]
Yes, this will be possible in 2.0 thanks to the new GraphQL API.
